diagnostic.md

```diff
@@ -7,40 +7,48 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git clone git@github.com:narichasavanorkejoyce/git-diagnostic.git
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+// Navigate to the folder containing the cloned Git repo
+cd git-diagnostic
```

Comment:
Hey Nari this is good but we were specifically looking for creating a new branch so that you can do work without affecting the master branch.